setup.py

```diff
@@ -0,0 +1,12 @@
+import setuptools
+
+setuptools.setup(
+       name = 'yahtzee_package',
+       version = '0.0.1',
+       author = 'Dana Ravestein',
+       author_email = 'dana.ravestein94@gmail.com',
+       python_requires = '>= 3.6',
+       packages = setuptools.find.packages()
+        
+)
+
```

Comment:
if you want you can 'link' the readme file in their directly:

https://packaging.python.org/tutorials/packaging-projects/

This is the way to do it, if you are interested: 

`import setuptools

with open("README.md", "r") as fh:
    long_description = fh.read()

setuptools.setup(
    name="example-pkg-YOUR-USERNAME-HERE", # Replace with your own username
    version="0.0.1",
    author="Example Author",
    author_email="author@example.com",
    description="A small example package",
    long_description=long_description,
    long_description_content_type="text/markdown",
    url="https://github.com/pypa/sampleproject",
    packages=setuptools.find_packages(),
    classifiers=[
        "Programming Language :: Python :: 3",
        "License :: OSI Approved :: MIT License",
        "Operating System :: OS Independent",
    ],
    python_requires='>=3.6',
)`